content/docs/addons-shallow-renderer.md

```diff
@@ -13,11 +13,12 @@ import ShallowRenderer from 'react-test-renderer/shallow'; // ES6
 var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
 ```
 
-## Overview {#overview}
+## Επισκόπηση {#overview}
 
-When writing unit tests for React, shallow rendering can be helpful. Shallow rendering lets you render a component "one level deep" and assert facts about what its render method returns, without worrying about the behavior of child components, which are not instantiated or rendered. This does not require a DOM.
+Κατά την διαδικασία εγγραφής unit tests για React, το shallow rendering μπορεί να είναι ιδιαίτερα χρήσιμο. To Shallow rendering σου επιτρέπει να κάνεις render ένα component "σε βάθος ενός επιπέδου"  και να κάνεις assert γεγονότα όπως τι επιστρέφει στο render μια method, χωρίς να ανασυχείς για την συμπεριφορά των εσωτερικών components της μεθόδου,
+τα οποία δεν αποδίδονται ή αναφέρονται. Αυτό δεν απαιτεί ένα DOM. 
```

Comment:
It would be better to keep parity with the english copy and merge this line with the above if we can.